_commands/basics/tail.md

```diff
@@ -0,0 +1,70 @@
+---
+---
+
+tail
+-------
+
+Tail displays the last part(s) of a file
+
+~~~ bash
+$ tail input.txt
+$ tail -f -n 15 input.txt
+~~~
+
+<!--more-->
+
+### Useful Options / Examples
+
+#### `tail input.txt`
+
+~~~ bash
+$ tail input.txt
+	}
+else {
+		int x1 =0;
+		int y1=0;
+		int gcd = inverse_helper(b%a, a, &x1, &y1);
+		*x = y1- (floor(b/a) *x1);
+		*y = x1;
+		return gcd;
+	}
+	return 1000;
+~~~
+
+##### Break it down
+* Shows the last 10 lines of input.txt
+
+#### `tail -f -n 5 input.txt`
+
+
+~~~ bash
+$ tail -f -n 5 input.txt
+		*x = y1- (floor(b/a) *x1);
+		*y = x1;
+		return gcd;
+	}
+	return 1000;
+[waiting for more]
+~~~
+
+##### Break it down
+* Shows the last 5 lines of input.txt. The -f command causes tail to not stop but rather waits for additional data to be added to file
```

Comment:
Grammar on line 51: "

Also, nitpick: The -f should be surrounded with backticks- as should every flag.